src/common/snippets/include/snippets/op/reduce.hpp

```diff
@@ -41,6 +41,12 @@ class ReduceSum : public ReduceBase {
     static std::set<ov::element::TypeVector> get_supported_precisions(const std::shared_ptr<ov::Node>& node) {
         return {{ov::element::f32}};
     }
+    /**
+     * @brief Creates ReduceSum operation, computes and sets input/output subtensors
```

Comment:
Minor:
`Creates ReduceSum operation, computes and sets subtensors to input/output PortDescriptors`
May be it's better 😃 

BTW, class `ReduceSum` has method `make_reduce_sum` - probably just `make` or `build` are simpler. But I think. a matter of taste 😃

Reply:
Done